actors/miner/src/expiration_queue.rs

```diff
@@ -69,11 +75,24 @@ impl ExpirationSet {
         Ok(())
     }
 
+    pub fn combine(&mut self, other: &Self) -> anyhow::Result<()> {
+        self.on_time_sectors |= &other.on_time_sectors;
+        self.proof_expiring_sectors |= &other.proof_expiring_sectors;
+        self.faulty_sectors |= &other.faulty_sectors;
+        self.on_time_pledge += &other.on_time_pledge;
+        self.active_power += &other.active_power;
+        self.faulty_power += &other.faulty_power;
+
+        self.validate_state()?;
+        Ok(())
+    }
+
     /// Removes sectors and power from the expiration set in place.
     pub fn remove(
         &mut self,
         on_time_sectors: &BitField,
         early_sectors: &BitField,
```

Comment:
It's confusing to keep the same name but with a different meaning.  Any problem with calling this parameter `proof_expiring_sectors`?

Reply:
It was because the `proof_expiring_sectors` was a late rename of the field, I missed this method.